weeks/week02/HW2_Task1.py

```diff
@@ -0,0 +1,30 @@
+my_list = [0,-12,8,-44,461]
+my_list_lenth = len(my_list)
+for i in range(my_list_lenth):
+    for j in range(my_list_lenth - i - 1):
+        if my_list[j] > my_list[j+1]:
+           my_list[j], my_list[j+1] = my_list[j+1], my_list[j]
+           print(my_list)
```

Comment:
I understood that you wanted to show the numbers being sorted for each interaction but ideally, it should be printed only after all computation (out of the for statement), unless it is required or wanted. It could be a function and the return would be the list sorted. See the same for all other exercises.